src/plugins/messenger/MessengerPreview/components/MessengerMediaTemplate/MessengerMediaTemplate.tsx

```diff
@@ -67,8 +80,26 @@ export const getMessengerMediaTemplate = ({
             }, 100);
         }
 
-        const { media_type, url, altText } = element as IFBMMediaTemplateUrlElement;
-        // TODO add buttons
+        const imageButtons = () => {
```

Comment:
should we not return null here if buttons is falsy? else you would return an empty div